tt_metal/llrt/tt_cluster.hpp

```diff
@@ -264,28 +265,28 @@ class Cluster {
 
     // There is one device driver per PCIe card. This map points id of the MMIO device points to the associated device
     // driver
-    std::unordered_map<chip_id_t, std::unique_ptr<tt_device>> mmio_device_id_to_driver_;
+    tt::IndexMap<std::unique_ptr<tt_device>> mmio_device_id_to_driver_;
```

Comment:
PR https://github.com/tenstorrent/tt-metal/pull/13949 will be merged soon and updates this so that there is one device driver for all devices detectd in a cluster (this will not need to be an IndexMap at that point)

Reply:
Sounds amazing!